src/main/java/frc/robot/subsystems/PDP.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.subsystems;
+
+
+import edu.wpi.first.wpilibj.PowerDistributionPanel;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import frc.robot.RobotMap;
+
+public class PDP extends Subsystem {
```

Comment:
I'm not sure how I feel about having things like the PDP (which to me, seem like sensors) as full-fledged `Subsystem`s. WPI describes `Subsystem`s as ["parts of your robot that are independently controlled"](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599735-simple-subsystems), which to me means this isn't the intended use case. The PDP isn't "independently controlled" - it's not really controlled at all.

That said, the PDP doesn't clearly "belong" to any system on the robot (like Talons etc.), so... I dunno. I'll give it more thought.

Reply:
Gave it thought! Make a general `Sensors` class that doesn't extend `Subsystem` with your utility methods for each sensor in it. Make it a singleton if you want as well.